ArduCopter/mode.h

```diff
@@ -1097,45 +1097,45 @@ class ModeGuided : public Mode {
-    bool set_pos_vel_neu_cm(const Vector3f& pos_neu_cm, const Vector3f& vel_neu_cms, bool use_yaw = false, float yaw_rad = 0.0, bool use_yaw_rate = false, float yaw_rate_rads = 0.0, bool yaw_relative = false);
-    bool set_pos_vel_accel_neu_cm(const Vector3f& pos_neu_cm, const Vector3f& vel_neu_cms, const Vector3f& accel_neu_cmss, bool use_yaw = false, float yaw_rad = 0.0, bool use_yaw_rate = false, float yaw_rate_rads = 0.0, bool yaw_relative = false);
+    void set_accel_NEU_mss(const Vector3f& accel_neu_mss, bool use_yaw = false, float yaw_rad = 0.0, bool use_yaw_rate = false, float yaw_rate_rads = 0.0, bool yaw_relative = false, bool log_request = true);
+    void set_vel_NEU_ms(const Vector3f& vel_neu_ms, bool use_yaw = false, float yaw_rad = 0.0, bool use_yaw_rate = false, float yaw_rate_rads = 0.0, bool yaw_relative = false, bool log_request = true);
+    void set_vel_accel_NEU_m(const Vector3f& vel_neu_ms, const Vector3f& accel_neu_mss, bool use_yaw = false, float yaw_rad = 0.0, bool use_yaw_rate = false, float yaw_rate_rads = 0.0, bool yaw_relative = false, bool log_request = true);
+    bool set_pos_vel_NEU_m(const Vector3f& pos_neu_m, const Vector3f& vel_neu_ms, bool use_yaw = false, float yaw_rad = 0.0, bool use_yaw_rate = false, float yaw_rate_rads = 0.0, bool yaw_relative = false);
+    bool set_pos_vel_accel_NEU_m(const Vector3f& pos_neu_m, const Vector3f& vel_neu_ms, const Vector3f& accel_neu_mss, bool use_yaw = false, float yaw_rad = 0.0, bool use_yaw_rate = false, float yaw_rate_rads = 0.0, bool yaw_relative = false);
 
     // get position, velocity and acceleration targets
-    const Vector3p& get_target_pos_neu_cm() const;
-    const Vector3f& get_target_vel_neu_cms() const;
-    const Vector3f& get_target_accel_neu_cmss() const;
+    const Vector3p& get_target_pos_NEU_m() const;
+    const Vector3f& get_target_vel_NEU_ms() const;
+    const Vector3f& get_target_accel_NEU_mss() const;
 
     // returns true if GUIDED_OPTIONS param suggests SET_ATTITUDE_TARGET's "thrust" field should be interpreted as thrust instead of climb rate
     bool set_attitude_target_provides_thrust() const;
-    bool stabilizing_pos_xy() const;
-    bool stabilizing_vel_xy() const;
+    bool stabilizing_pos_NE() const;
+    bool stabilizing_vel_NE() const;
     bool use_wpnav_for_position_control() const;
 
     void limit_clear();
     void limit_init_time_and_pos();
-    void limit_set(uint32_t timeout_ms, float alt_min_cm, float alt_max_cm, float horiz_max_cm);
+    void limit_set(uint32_t timeout_ms, float alt_min_m, float alt_max_m, float horiz_max_m);
     bool limit_check();
 
     bool is_taking_off() const override;
     
-    bool set_speed_xy_cms(float speed_xy_cms) override;
-    bool set_speed_up_cms(float speed_up_cms) override;
-    bool set_speed_down_cms(float speed_down_cms) override;
+    bool set_speed_NE_ms(float speed_xy_ms) override;
```

Comment:
I guess we should rename the argument to also be, "speed_NE_ms" or "speed_ne_ms"?